src/plugins/plugin.legend.js

```diff
@@ -106,6 +107,12 @@ var Legend = Element.extend({
 		// Contains hit boxes for each dataset (in dataset order)
 		this.legendHitBoxes = [];
 
+		// Contains the currently hovered legend item
```

Comment:
I'd remove this comment because it doesn't really add anything you don't get from the variable name already

If it was was necessary to keep then you should combine it with the lines below like: https://github.com/chartjs/Chart.js/blob/d22c535c1001b5e6e90ebcbacd9a87b8cd273d7d/src/plugins/plugin.legend.js#L88